projects/observability/src/shared/components/cartesian/chart.ts

```diff
@@ -122,6 +122,11 @@ export interface Axis {
    * Maximum value of the axis. If unset, defaults to maximum value of provided data.
    */
   max?: number;
+
+  /**
+   * Determine the distance in days between each tick.
+   */
+  tickDaysDistance?: number;
```

Comment:
hmm, so does this depend on the range of time in data points? What if all data points are within the last hour and `tickDaysDistance `is set to 4? How many ticks would it end up generating?

Reply:
I understand that this logic only works with periods of time relative to days, to adjust a change of that type you should go back to the variable of type TimeInterval and pass the specific function directly to it.